### ISSUE TITLE
Pre-trained models?

### ISSUE THREAD

**Issue by andreaswallberg** [muriloHoracio/TERL#11]
Dear @muriloHoracio & @brunocontrerasmoreira ,

I am looking to try out your software to annotate a repeat library from RepeatModeler for a non-model invertebrate.

Can I use the models in Models/ds*_model.zip, and if so, which one do you recommend?

**Comment by muriloHoracio** [comment]
Hello @andreaswallberg,

Thank you for trying out TERL. The models in Models/ were trained as follows:

- Data set 1 (DS1) was trained based on RepBase sequences to classify the following superfamilies: Copia, Gypsy, Bel-Pao, ERV, L1, Tc1-Mariner, and hAT.
- DS2 was also trained based on RepBase to classify the sequences into Class II, LTR, and LINE.
- DS3 was trained with sequences from 7 databases and classifies sequences into Copia, Gypsy, Bel-Pao, ERV, L1, SINE, Tc1-Mariner, hAT, Mutator, PIF-Harbinger, and En/Spm - CACTA.
- DS4 was also trained with sequences from 7 databases, but it classifies sequences into Class II, LTR, LINE, and SINE.
- DS5 was trained with sequences from RepBase and tested with sequences from the other databases. It can classify sequences into Class II, LTR, and LINE.

All models were trained with a class of non-TE sequences that were generated by shuffling the sequences to simulate noise.

Basically, DS1 and DS3 are for superfamily classification, whereas DS2, DS4, and DS5 are for order.

In order to obtain better results, I would suggest training TERL with sequences more related to your object of study. It should increase the performance of the model.
If this is not an option for you, I would suggest the Models/ds3_model.zip, since it was trained with sequences from all databases and provides the classification into superfamily.
If Models/ds3_model.zip performs poorly, I would try the Models/ds1_model.zip.

We are working to improve TERL and more models can be added to our library.

Hope it helps and you can use TERL with success!
Please let me know if you need any further help.

**Comment by xiekunwhy** [comment]
Hi,

There is no helitrons in all trained models?

Best,
Kun
